src/styles/vuesax-overrides.scss

```diff
@@ -1,3 +1,30 @@
+// Colors
+:root {
+    --vs-primary: 0, 0, 0;
+    --vs-success: 23, 201, 100;
+    --vs-danger: 242, 19, 93;
+    --vs-warn: 254, 130, 0;
+    --vs-dark: 36, 33, 69;
+}
+
+// Common
 body {
     font-family: 'Poppins-Regular';
```

Comment:
@PVillasana Why, are not you using the font variable?

Reply:
@Andresws12 You are telling me to use the library variable to override the font?

Reply:
@PVillasana No, You declared a new variable and you didn't use this

Reply:
![image](https://user-images.githubusercontent.com/47737142/119232434-92711280-bb25-11eb-8b25-643b9bbbf45a.png)

Reply:
@Andresws12 Ok, you refereed to the fallback fonts variables